Doc/includes/wasm-ios-notavail.rst

```diff
@@ -0,0 +1,8 @@
+.. include for modules that don't work on WASM or iOS
+
+.. availability:: not Emscripten, not WASI, not iOS.
```

Comment:
Re: https://github.com/python/cpython/issues/115977, we can remove all the Emscritpen mentions.

At least don't add new ones. 

Optionally, remove the others in edited lines, but fine to leave those be if you'd prefer to keep this PR focused.

Reply:
I've purged *almost* all the references to Emscripten; there are some remaining references in historical release notes, plus in `library/sys.rst` and `library/intro.rst`. The remaining references are for behavior that still *exists* in the codebase, even if Emscripten isn't covered by Tier 3. AFAICT this is analogous to methods for Android support that are *documented*, even though Android isn't Tier 3 (... yet 😄). Where the text referred to specific platform descriptors, I've simplified to just say "on WebAssembly" (or similar).